tests/e2e/config.yaml

```diff
@@ -0,0 +1,3 @@
+appName: ingress-nginx
+repoName: nginx-ingress-controller-app
```

Comment:
```suggestion
repoName: ingress-nginx-app
```
😉

Reply:
Did we rename the repo at some point? 🤨  I have it checked out as 
```
/Users/marcus/Code/GiantSwarm/nginx-ingress-controller-app
```

Reply:
But you took a look at where you filed your PR at, didn't you? 😬 It is like this for quite a long time now.

<img width="339" alt="Bildschirmfoto 2024-02-24 um 10 55 42" src="https://github.com/giantswarm/ingress-nginx-app/assets/935479/5ba45f66-fb47-4c01-88bd-04725fa7b610">

Reply:
Yeah. I realised after you commented. 😅 I must have checked this out to my machine a long time ago